### ISSUE TITLE
scene does not appear after Unity splash screen when building Unity project on Windows

### ISSUE THREAD

**Issue by cvan** [mozilla/unity-webvr-export#178]
I think most (all?) of us have been building the project in Unity on macOS and pointing Firefox on our VR computers to a local/ngrok URL.

I upgraded Unity (to the latest, Unity `2017.3.1f1` Personal Edition) on one of my Windows machines, and I couldn't get a build that loads properly. Generating a build from Unity on macOS still seems to work though (ignoring the terrain issue, which I've filed for in #177).

<hr>

screenshots below:

![image](https://user-images.githubusercontent.com/203725/36779047-85034248-1c23-11e8-9eb2-7ab9c43e89c7.png)
![image](https://user-images.githubusercontent.com/203725/36779049-873771c4-1c23-11e8-8a67-b5b04db993ec.png)
![image](https://user-images.githubusercontent.com/203725/36779060-8e3dbea6-1c23-11e8-868a-89595e03f099.png)
![image](https://user-images.githubusercontent.com/203725/36779068-929dbee2-1c23-11e8-826d-5a3d7a8400ce.png)

and the scene gets stuck there!

<hr>

@caseyyee @delapuente: can y'all comment here with the platforms (macOS or Windows) versions and workflows you have been building, testing, and developing with? that might help us get to the bottom of the inconsistencies I'm seeing here. thanks!

**Comment by delapuente** [comment]
I'm using Windows.

**Comment by delapuente** [comment]
Can not reproduce. @cvan @caseyyee can you check this again?

**Comment by cvan** [comment]
On a fresh clone, opening the project in Unity:

![image](https://user-images.githubusercontent.com/203725/37639174-92fe8e0c-2bcd-11e8-85fb-d76f757f38c7.png)

And in Firefox:

![image](https://user-images.githubusercontent.com/203725/37639184-9d64c9ba-2bcd-11e8-8c5c-87eebd67cd75.png)

Notice the **terrain is missing**. Is there another issue I should file? Or shall I change the title of this issue?

I'm on Unity 2017.3.1f1. I haven't upgraded yet, because I wanted to first see if you, @delapuente or @caseyyee, can reproduce before I upgrade. Let me know.

Thanks!

**Comment by caseyyee** [comment]
@cvan I did test the terrain issue and it seems like it could be a _path_ problem between macOS and windows.   If you go into `Assets/Models/Desert Environment` and double-click the `.blend` file it should resolve it?   Give it a try and let me know.  You may need blender installed.

Looking at this, it may also be a good idea for me to save the model as some other format than blender (like obj?).

**Comment by cvan** [comment]
ah, good call! you're right:

![image](https://user-images.githubusercontent.com/203725/37747036-54f0d548-2d3a-11e8-86b7-18767564c447.png)

**Comment by cvan** [comment]
this is no longer an issue. the terrain issue I'm handling separately in PR #212. 🏜

**Comment by delapuente** [comment]
Let's leave it open. It will be closed when merging the PR automatically. This way we track what remains accurately and the bugs are not marked as closed until we get the PR merged.

**Comment by cvan** [comment]
what I was not super clear about (and should have filed a separate issue for): the problem I experienced when I filed this report and described in the first comment, we realised the problem was no longer reproducible by anyone else, this issue report morphed into an issue about Blender vs. OBJ models in Unity.

1. change the title of this issue, or
2. close this one and file a new issue for "Blender terrain errors in Unity without Blender installed"
3. either way, PR #212 addresses the latter issue

whichever makes most sense to y'all. many thanks for testing, investigating, and solving this together =)
